src/Queue/QueueHandler.php

```diff
@@ -124,7 +124,7 @@ protected function marshalJob(SqsRecord $sqsRecord): SqsJob
             $this->sqs,
             $message,
             $this->connection,
-            $this->queueName,
+            $sqsRecord->getQueueName() ?? $this->queueName,
```

Comment:
Oh this is good, thank you!

The new method will either exist (latest Bref version) or not, so I don't think the `??` needs to exist:

```suggestion
            $sqsRecord->getQueueName(),
```

Which also means you'll need to bump the required Bref version in `composer.json` to https://github.com/brefphp/bref/releases/tag/2.1.8

Reply:
Done.